providers/org/repec/subject-mapping.json

```diff
@@ -0,0 +1,32 @@
+{
+    "Agricultural economics": ["Q00", "Q01", "Q02", "Q1", "Q10", "Q11", "Q12", "Q13", "Q14", "Q15", "Q19", "Q17", "Q18", "Q19", "Q2", "Q20", "Q21", "Q22", "Q23", "Q24", "Q25", "Q26", "Q27", "Q28", "Q29", "Q3", "Q30", "Q31", "Q32", "Q33", "Q34", "Q35", "Q37", "Q38", "Q39", "Q4", "Q40", "Q41", "Q42", "Q43", "Q47", "Q48", "Q49", "Q5", "Q50", "Q51", "Q52", "Q53", "Q54", "Q55", "Q5", "Q56", "Q57", "Q58", "Q59"],
```

Comment:
Subject codes like this make me worry we'll run into synonym collisions sooner than later. We should probably make subject mappings source-specific...